ajax/libs/simplebar/package.json

```diff
@@ -19,7 +19,7 @@
     "target": "git://github.com/grsmto/simplebar.git",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "packages/simplebar/dist",
```

Comment:
Hi, I can't find `dist` folder after 2.6.1 on GitHub, how about using npm?

Reply:
Yeah you are right, the `dist` folder is not tracked by Github, it's only in the distributed npm package.
What should I do, should I commit the `dist`folder or should I point to the unminified `src` folder? (I saw cdnjs supports minifying).

Reply:
You can use npm auto-update config 😃  more details [here](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).
In short, replace the current git auto-update with
```json
"npmName": "simplebar",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/!(*.esm.*)"
      ]
    }
  ]
```
will work :)

Reply:
@extend1994 could you take a look at the latest CI build? I don't understand why it's failing :/
Seems like I'm following the pattern you described.
Thanks!

Reply:
@Grsmto Sure :)  you only need https://github.com/cdnjs/cdnjs/blob/835d96fa5d9707f02f5f1d0b57f980f7ce4aa359/ajax/libs/simplebar/package.json#L18-L26 without L17 & L27. Don't forget to have them be indented in 2 spaces.